src/Blocks/BlockOre.h

```diff
@@ -50,6 +50,11 @@ class cBlockOreHandler :
 				a_Pickups.push_back(cItem(E_ITEM_COAL));
 				break;
 			}
+			case E_BLOCK_QUARTZ_ORE:
+			{
+				a_Pickups.push_back(cItem(E_ITEM_QUARTZ));
+				break;
+		        }
```

Comment:
I think I got them all. Is this the way you need it to be now?

On Wed, Apr 22, 2015 at 2:17 PM, NiLSPACE <notifications@github.com> wrote:

> In src/Blocks/BlockOre.h
> <https://github.com/mc-server/MCServer/pull/1888#discussion_r28866397>:
>
> > @@ -50,6 +50,11 @@ class cBlockOreHandler :
> >  				a_Pickups.push_back(cItem(E_ITEM_COAL));
> >  				break;
> >  			}
> > +			case E_BLOCK_QUARTZ_ORE:
> > +			{
> > +				a_Pickups.push_back(cItem(E_ITEM_QUARTZ));
> > +				break;
> > +		        }
>
> You're using a combination of spaces and tabs here. Could you change it to
> tabs?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mc-server/MCServer/pull/1888/files#r28866397>.
>